src/components/Hero.style.js

```diff
@@ -0,0 +1,75 @@
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+
+export const DivStyled = styled.div`
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+`;
+
+export const H1Styled = styled.h1`
+  font-family: Bitter;
+  font-size: 38px;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  text-align: center;
+  color: var(--gray-dark);
+  margin-top: 27px;
+  margin-bottom: 7px;
+}
+`;
+
+export const H2Styled = styled.h2`
```

Comment:
Something is off here by 1px I guess

![image](https://user-images.githubusercontent.com/9722792/77858713-787bbe80-7205-11ea-8be9-7411b2781673.png)